examples/kit-nextjs-skate-park/src/components/container/Container.tsx

```diff
@@ -1,6 +1,7 @@
-import { Placeholder } from '@sitecore-content-sdk/nextjs';
+import Placeholder from 'components/content-sdk/Placeholder';
```

Comment:
Can you, please, sort imports properly in all the `components/**` files? local imports should be placed below while imports from npm packages should be at the top